<launch>

  <!-- MCPTAM Server node -->
    <node name="mcptam_server" pkg="mcptam" type="mcptam_server" clear_params="true" output="screen">
    <rosparam command="load" file="$(find mcptam)/groups/$(arg group_name).yaml" />
    
    <param name="get_pose_separately" type="bool" value="true" />
    <param name="mm_init_point_mode" type="string" value="stereo" /> 
    <param name="camera_pose_file" type="string" value="$(find mcptam)/poses/poses.dat" />
  
    <remap from="modify_map" to="mcptam_client/modify_map"/>
    <remap from="tracker_state" to="mcptam_client/tracker_state" />
    <remap from="tracker_small_image/compressed" to="mcptam_client/tracker_small_image/compressed" />
    <remap from="reset" to="mcptam_client/reset" />
    <remap from="init" to="mcptam_client/init" />
    <remap from="system_info" to="mcptam_client/system_info" />
    <param name="image_transport" type="string" value="compressed"/> 
    <param name="kf_distance_mean_diff_fraction" type="double" value="0.5" /> 
  </node>
 
<node pkg="tf" type="static_transform_publisher" name="world_tf" args="0 0 0 -0.5 0.5 -0.5 0.5 world vision_world 100" />

</launch>

